Update aliasing-api.md to reflect new GET endpoint (#67)
amplitude/amplitude-dev-center: docs/analytics/apis/aliasing-api.md

        **Result**

        The users "63629@hmail.com" and "12345@hmail.com" are unmapped from any global user IDs they're mapped to. 

## Get existing user mappings 

Get the list of mappings that involve the provided user ids. 

> NOTE: This is a different url, https://amplitude.com/api/2/usermap, than the map and unmap endpoints above. 

Reviewer: ```suggestion
!!! info
    User mappings uses a different URL than the map and unmap endpoints above, `https://amplitude.com/api/2/usermap`.
```

### Query parameters

|<div class="big-column">Name</div>|Description|
|----|-----------|
|`api_key`| Required. String. an API Key for any project in your organization.|
|`secret_key`| Required. String. an API Secret Key for any project in your organization.|
|`user_ids`| Required. A list of user ids. Minimum 1 user id, maximum 100 user ids in a single request. |

> 

Reviewer: ```suggestion
```

=== "python" 

> ```python

Reviewer: ```suggestion

```python
```

    user_ids = ["<REPLACE ME WITH LIST OF USER IDS>"]

    request_data = {
        "user_ids": user_ids,
    }
    url = 'https://amplitude.com/api/2/usermap'

    API_KEY = "REPLACE_ME"
    SECRET_KEY = "REPLACE_ME"
    result = requests.get(url, auth=HTTPBasicAuth(API_KEY, SECRET_KEY), data=request_data)
    if result.status_code != 200:
        raise AssertionError(
            "Request failed. Error: code - %s, text - %s" % (result.status_code, result.text))
    else:
        print(result.json())
```

### Response

The response for a POST request contains these fields:

| <div class="big-column">Name</div> | Description                                                                                               |
| ---------------------------------- | --------------------------------------------------------------------------------------------------------- |
| `user id`                          | A user id in the request list.                                                                            |

> | `mappings`                         | The mappings associated with this user id. See next section.  Empty if the requested user isn't found     |

Reviewer: ```suggestion
| `mappings`                         | The mappings associated with this user id. See next section. Empty if the requested user isn't found     |
```
